shell/platform/windows/flutter_windows_view.cc

```diff
@@ -444,10 +535,34 @@ bool FlutterWindowsView::SwapBuffers() {
   }
 }
 
+bool FlutterWindowsView::Present(const void* allocation, size_t row_bytes, size_t height) {
```

Comment:
I find it a little bit weird having a present method on the view that is only needed / used by the software path.  At a minimum this should be named to be software only but I do wonder if the refactor you've done on the render config could be slightly improved to avoid this.

Reply:
We could probably do the blitting in the callback directly, but I'd generally prefer to have that functionality in a method on the class itself to more easily read the code. I've renamed this to `PresentSoftwareBitmap` accordingly.